Makefile

```
@@ -1,6 +1,9 @@
 CURRENT_COMMIT := $(shell git rev-parse HEAD)
 
-build-dev: build-ui
+build-only-dev: build-ui
+	node_modules/.bin/antora generate only-dev-antora-playbook.yml
+
+build-local: build-ui
 	node_modules/.bin/antora generate local-antora-playbook.yml
```

```suggestion
build-local: build-ui
	node_modules/.bin/antora generate local-antora-playbook.yml

build-only-dev: build-ui
	node_modules/.bin/antora generate only-dev-antora-playbook.yml
```

either the default recipe needs to be the `local` one, or the `.github/build.yml` needs to be updated to call the `make build-local` explicitly.

README.adoc:25 also just mentions "make", also would need to be updated. 

I'm not sure whether we want the local or only-dev one to be the default. I think i#d like "only-dev" to be the special case and local the default, hmm. Not sure.

thanks for all the other changes, look great!

---

Haha, I though of defaulting to only the dev versions by default.
But I don't have a strong opinion, so picking yours :)